applications/sdp/gpio/boards/nrf54l15dk_nrf54l15_cpuflpr_icbmsg.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	ipc {
+		ipc0: ipc0 {
```

Comment:
would like some more information on this.

Other boards, such as nrf54h20dk, nrf5340dk, thingy53, are defining default ipc nodes, which can then have the status changed from `disabled` to `okay`.

Any reason the nrf54l15dk doesn't define a default (perhaps disabled) ipc node ?

Examples upstream:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-ipc_conf.dtsi
https://github.com/zephyrproject-rtos/zephyr/blob/1726443d9d2584bc6adb7fd08d63a5fc645ce09a/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts#L159

Or at SoC level, like the nrf5340:
https://github.com/zephyrproject-rtos/zephyr/blob/1726443d9d2584bc6adb7fd08d63a5fc645ce09a/dts/arm/nordic/nrf5340_cpunet.dtsi#L349-L350
https://github.com/zephyrproject-rtos/zephyr/blob/1726443d9d2584bc6adb7fd08d63a5fc645ce09a/dts/arm/nordic/nrf5340_cpuapp.dtsi#L110-L111

@gmarull perhaps you have some input regarding this.

Reply:
My understanding is that H20 has established strict resource allocation scheme (in this case VEVIF/BELLBOARD channels and shared memory) due to complexity and security architecture.
Meanwhile, L15 has only two CPUs with access to all resources by default, so it is up to developer to decide what to use. It is much closer to 53's case - I'm not aware what was the reasoning behind those defaults.
@hubertmis can you comment?

Reply:
> Meanwhile, L15 has only two CPUs with access to all resources by default, so it is up to developer to decide what to use. 

but in current case, you are asking every sample to create the overlay and decide on the ram allocation.
Which in most case will result in users looking for a template like this file, and start copying that around.

If the board provide a sane set of defaults, then a sample should be a to just set the node to enabled and not worry about other parts.
Users with special need can still adjust memory allocation as they find fit.

Reply:
@tejlmand, can we change that in a follow-up PR? I wouldn't want to change it here since this PR only adds another backend to eGPIO. If I were to move the definition of the IPC node to board or soc dts, I would also have to modify overlays for other eGPIO backends (mbox and icmsg), which is not the topic of this PR.
@hubertmis, can you share your opinion on where the IPC definition should be placed?

Reply:
Those discussions are out of scope of this PR - two other snippets already use this method. This PR only adds a new configuration.
If we want to change this, then a discussion and a PR reworking (not for 2.8) those snippets needs to be conducted.

Reply:
I  missed the request for my comment. We need to define a unified scheme how to easily allocate memory and IPC resources for VPRs. We need to consider two scenarios:

1. All samples using VPRs would like to allocate almost the same resources for VPRs
2. All samples not using VPRs would like to allocate all resources for App

To easily meet both requirements, we should define one (probably the one with more samples) in the board files, and have a common way to apply overlays for the other one (snippets?). What is the default and what is applied should be unified for all nRF54 (L and H). And the solution for applying changes also should be unified (if we choose snippets, use snippets for 54L and H, reuse naming, etc)